omero/users/cli/import.txt

```diff
@@ -15,91 +15,136 @@ Import command
 .. program:: omero import
 
 :omerocmd:`import` is probably the first command many users will want to use.
```

Comment:
Given that this is now on a page of its own this first sentence seems a little out of place. I'd probably just drop the entire sentence.